### PR TITLE
[docs] Update the roadmap

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**

### CODE REVIEW

File: docs/articles/roadmap/README.md [hunk 1]
```diff
@@ -15,18 +15,16 @@ Please vote on the features you want us to implement first to help us prioritize
 
 Feature                                                                                           | Status
 ------------------------------------------------------------------------------------------------- | -------------------
-[Visual Regression Testing](https://github.com/DevExpress/testcafe/issues/1207)                   | Under Consideration
-[Testing in Multiple Browser Windows](https://github.com/DevExpress/testcafe/issues/912)          | Under Consideration
+[Testing in Multiple Browser Windows](https://github.com/DevExpress/testcafe/issues/912)          | Planned
+[HTTP API Testing](https://github.com/DevExpress/testcafe/issues/4288)                            | Planned
```
Comment (VasilyStrelyaev) [inline docs/articles/roadmap/README.md]: Is it really `HTTP API Testing` - not `REST API Testing` or something?

Comment (kirovboris) [reply docs/articles/roadmap/README.md]: REST API testing is a subset of API Testing. I think API testing is the more common term that HTTP API testing..

File: docs/articles/roadmap/README.md [hunk 2]
```diff
@@ -15,18 +15,16 @@ Please vote on the features you want us to implement first to help us prioritize
...
 ## API
 
 Feature                                                                                           | Status
 ------------------------------------------------------------------------------------------------- | -------------------
 [Configuration File](https://github.com/DevExpress/testcafe/issues/3131)                          | **Implemented**
-[Screenshots of the Entire Page](https://github.com/DevExpress/testcafe/issues/1520)              | Under Consideration
-[Custom Actions](https://github.com/DevExpress/testcafe/issues/1535)                              | Under Consideration
-[Conditional Test Skip](https://github.com/DevExpress/testcafe/issues/1626)                       | Under Consideration
-[Browser Information in Test Code](https://github.com/DevExpress/testcafe/issues/481)             | Under Consideration
+[Screenshots of the Entire Page](https://github.com/DevExpress/testcafe/issues/1520)              | Planned
```
Comment (helen-dikareva) [inline docs/articles/roadmap/README.md]: **In Progress**

File: docs/articles/roadmap/README.md [hunk 3]
```diff
@@ -15,18 +15,16 @@ Please vote on the features you want us to implement first to help us prioritize
 
 Feature                                                                                           | Status
 ------------------------------------------------------------------------------------------------- | -------------------
-[Visual Regression Testing](https://github.com/DevExpress/testcafe/issues/1207)                   | Under Consideration
-[Testing in Multiple Browser Windows](https://github.com/DevExpress/testcafe/issues/912)          | Under Consideration
+[Testing in Multiple Browser Windows](https://github.com/DevExpress/testcafe/issues/912)          | Planned
```
Comment (helen-dikareva) [inline docs/articles/roadmap/README.md]: Can we say **In Progress** progress already?

File: docs/articles/roadmap/README.md (hunk 1 above)
Comment (miherlosev) [reply docs/articles/roadmap/README.md]: API testing is very common term included load, security and etc. (https://en.wikipedia.org/wiki/API_testing).
In my opinion, HTTP API testing or Web API testing is best variant.

Comment (kirovboris) [reply docs/articles/roadmap/README.md]: Of course it is, but in other products it named just API testing.

Comment (miherlosev) [reply docs/articles/roadmap/README.md]: Ok.

File: docs/articles/roadmap/README.md (hunk 3 above)
Comment (VasilyStrelyaev) [reply docs/articles/roadmap/README.md]: @kirovboris

Comment (kirovboris) [reply docs/articles/roadmap/README.md]: Sure

**Pull request merged**